### DISCUSSION TITLE
What is a base package?

### DISCUSSION THREAD

**Discussion by mon-jai** [msys2/MSYS2-packages#2878] [Q&A] [answered]
Hi.

I found [`mingw-w64-clang`](https://packages.msys2.org/base/mingw-w64-clang) but unable to install it with `pacman -S mingw-w64-clang`.

What is a base package? How is it different from usual pacakges? How can I install them?

**Comment by Biswa96** [comment, marked as answer]
It's just a name to group the packages together.

**Reply by mon-jai** [reply]
Are they anything but just a name? Can they be installed?

**Reply by lazka** [reply]
They are just a name

**Comment by goyalyashpal** [comment]
also see:

https://github.com/msys2/MSYS2-packages/issues/3457#issuecomment-1366980203
https://github.com/msys2/MSYS2-packages/issues/3457#issuecomment-1397609668

use pacboy to "avoid writing long package names"
